homebrew_challenge/src/App.jsx

```diff
@@ -107,23 +108,7 @@ const App = () => {
 
   return (
     <div className="App">
-      <header>
-        <div className="heading-container">
-          <div className="heading">
-            <img
-              onClick={() => setCurrentPage(PAGE_OVERVIEW)}
-              id="logo"
-              src="STALogo.png"
-              alt="Scottish Tech Army Logo"
-            />
-          </div>
-          <div className="heading heading-title">
-            <h1>Scottish COVID-19 Statistics</h1>
-            <h2>{pageTitle()}</h2>
-          </div>
-        </div>
-      </header>
-
+      <Header setCurrentPage={setCurrentPage} pageTitle={pageTitle} />
```

Comment:
There's going to be a conflict with the navbar changes here. Last one to merge will feel the pain... :)